glusterapi/peer.py

```diff
@@ -2,32 +2,23 @@
 
 
 class PeerApis(BaseAPI):
-    def peer_probe(self, host):
+    def peer_add(self, host):
         """
-        Gluster Peer Probe
+        Gluster Peer Add
 
         :param host: (string) Hostname or IP
         :raises: GlusterApiError on failure
         """
-        pass
+        return self._handle_request(self._post, 201, "/v1/peers", host)
```

Comment:
cannot pass host as  a request body (include zone and metadata in request, you need to send it like below
```
req = dict()
        req['addresses'] = []
        req['addresses'].append(host)
        if metadata is None:
            metadata = dict()
        req['metadata'] = metadata
        req['zone'] = zone
        return self._handle_request(self._post, 201, "/v1/peers",
                                    json.dumps(req))
```